apps/startrek/src/__tests__/pages/Web.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import Web from "@src/pages";
```

Comment:
Name this file Web and Galaxy for the galaxy app?

Reply:
Done done. ✅